x-pack/platform/plugins/shared/entity_manager/server/lib/auth/api_key/saved_object.ts

```diff
@@ -6,12 +6,17 @@
  */
 
 import { SavedObjectsErrorHelpers, SavedObjectsClientContract } from '@kbn/core/server';
+import { v5 as uuidv5 } from 'uuid';
 import { EntityDiscoveryApiKeyType } from '../../../saved_objects';
 import { EntityManagerServerSetup } from '../../../types';
 import { EntityDiscoveryAPIKey } from './api_key';
 
 const ENTITY_DISCOVERY_API_KEY_SO_ID = '19540C97-E35C-485B-8566-FB86EC8455E4';
 
+export const getSpaceAwareEntityDiscoverySavedObjectId = (space: string) => {
```

Comment:
Not essential. But because we have the ID generation in here, should we also abstract and move our access methods into here to keep things in one place?